lib/facade_converter.ts

```diff
@@ -93,21 +118,31 @@ export class FacadeConverter extends base.TranspilerBase {
   }
 
   private getFileAndName(symbol: ts.Symbol): {fileName: string, qname: string} {
-    while (symbol.flags & ts.SymbolFlags.Alias) symbol = this.tc.getAliasedSymbol(symbol);
-    if (!symbol.valueDeclaration) return null;
+    while (symbol.flags & ts.SymbolFlags.Alias) {
+      symbol = this.tc.getAliasedSymbol(symbol);
+    }
+    let decl = symbol.valueDeclaration;
+    if (!decl) {
+      // Martin: I started getting a null return value from here, which I suppose didn't happen with
```

Comment:
I got the `valueDeclaration` from a code snippet from @vladima, I think. My understanding is that `valueDeclaration` is the first place where the `Symbol` actually gets defined as a value. If you have a pure declaration, i.e. `declare interface Foo {}`, you just don't have an actual value. Maybe that's the issue? Only a theory though.